### PR TITLE
network: modify unsafe libc function

### PR DESCRIPTION
**Pull request by pillip8282**
change sprintf which is unsafe to snprintf

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [dbc4b67200ec7a6bc697a53cb8782b31157f5d1f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [dbc4b67200ec7a6bc697a53cb8782b31157f5d1f] - Code Rule Check OK.

File: lib/libc/net/lib_inetntop.c [hunk 1]
```diff
@@ -114,7 +114,7 @@ static int inet_ipv4_ntop(FAR const void *src, FAR char *dest, socklen_t size)
 	}
 
 	ptr = (FAR uint8_t *)src;
-	sprintf(dest, "%u.%u.%u.%u", ptr[0], ptr[1], ptr[2], ptr[3]);
+	snprintf(dest, INET_ADDRSTRLEN, "%u.%u.%u.%u", ptr[0], ptr[1], ptr[2], ptr[3]);
```
Comment (sangwon03) [inline lib/libc/net/lib_inetntop.c]: what is the correct buffer size?  "socklen_t size" or "INET_ADDRSTRLEN"

Comment (pillip8282) [reply lib/libc/net/lib_inetntop.c]: yes it's correct. refer below description
*   dest - The dest argument points to a buffer where the function stores
 *          the resulting text string; it shall not be NULL.
 *   size - The size argument specifies the size of this buffer, which must
 *          be large enough to hold the text string (INET_ADDRSTRLEN
 *          characters for IPv4, INET6_ADDRSTRLEN characters for IPv6).

**Pull request merged**